test/tigrc/segfault-test

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+. libtest.sh
+
+test_todo "Segfault without 'set reference-format' in tigrc"
```

Comment:
It's a real tig bug, though I haven't looked into the cause.

My motivation is that TODO test PRs are more substantive than issue discussions.